core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowResource.scala

```diff
@@ -453,7 +499,6 @@ class WorkflowResource extends LazyLogging {
         .execute()
     }
 
-    //TODO: copy the environment as well
```

Comment:
why remove this TODO? is it not relevant any more?

Reply:
Yes, because we have already removed the environment-related database.